### ISSUE TITLE
 model.train(np.asarray(Training_Data), np.asarray(Labels)) cv2.error: OpenCV(4.6.0) D:\a\opencv-python\opencv-python\opencv_contrib\modules\face\src\lbph_faces.cpp:362: error: (-210:Unsupported format or combination of formats) Empty training data was given. You'll need more than one sample to learn a model. in function 'cv::face::LBPH::train'

### ISSUE THREAD

**Issue by ranjeetkumar0703** [codingforentrepreneurs/OpenCV-Python-Series#28]
model.train(np.asarray(Training_Data), np.asarray(Labels))
cv2.error: OpenCV(4.6.0) D:\a\opencv-python\opencv-python\opencv_contrib\modules\face\src\lbph_faces.cpp:362: error: (-210:Unsupported format or combination of formats) Empty training data was given. You'll need more than one sample to learn a model. in function 'cv::face::LBPH::train'
